.github/workflows/static_analysis.yaml

```diff
@@ -52,6 +52,8 @@ jobs:
                     - "--noImplicitAny"
         steps:
             - uses: actions/checkout@v3
+              with:
+                ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
```suggestion
                  ref: ${{ github.event.pull_request.head.sha }}
```

Reply:
Oh, well this is what happens when committing from the web UI in-between two other tasks. 🤦

Thanks.